### ISSUE TITLE
Not announce by screenreader original reply

### ISSUE THREAD

**Issue by AlexandrKozlovskiy** [UnigramDev/Unigram#3035]
Hello. When blind user press tab key on a focused message,which is reply on some message,original message,on which  was reply,not announce by screenreader. @FrayxRulez you know about this issue but not fix it in beta version,so i report it here,because nobody done it before. Thanks for future fixing of issue.

**Comment by makisukurisu** [comment]
Hello, please, do not use Github Issues to report bugs.

Use https://bugs.telegram.org instead.

**Comment by AlexandrKozlovskiy** [comment]
@makisukurisu Ok,why in this case I can report issues for this repo?

**Comment by MarcAnt01** [comment]
@AlexandrKozlovskiy bugs are tracked in the place where @makisukurisu pointed you towards. We made this decision, in order to have a single place to collect bugs and feature requests, if you intend to report bugs/ make feature requests, do it there, as it's been written in big characters in GitHub before users try to open issues

**Comment by MarcAnt01** [comment]
By the way in this specific case, now there is no need to create another report, since the issue has been fixed

**Comment by AlexandrKozlovskiy** [comment]
@MarcAnt01 Thank you very much.
